tests/integration/tier0/check-cve-2022-1662/main.fmf

```diff
@@ -1,6 +1,23 @@
-summary: check cve-2022-1662 is fixed
+summary: Verify the CVE fixes
+description: |
+    2022-1662 CVE fixes leaking password and activation key to the command line when passing the value to subscription manager.
```

Comment:
The format is a unified thing in the industry https://cve.circl.lu/cve/CVE-2022-1662
```suggestion
    CVE-2022-1662 fixes leaking password and activation key to the command line when passing the value to subscription manager.
```